### PR TITLE
(12249) Implement advanced VPC attachment feature in TGW Orchestrator

### PR DESCRIPTION
**Pull request by zhiqiangjin**

### CODE REVIEW

File: test-infra/cmdExportOutput.sh [hunk 1]
```diff
@@ -192,7 +192,7 @@ echo "SKIP_GATEWAY_AWS=$SKIP_GATEWAY_AWS"
 echo "SKIP_GATEWAY_GCP=$SKIP_GATEWAY_GCP"
 echo "SKIP_GATEWAY_OCI=$SKIP_GATEWAY_OCI"
 echo "SKIP_GATEWAY_AWSGOV=$SKIP_GATEWAY_AWSGOV"
-echo "SKIP_GEO_VPN=$SKIP_GEO_VPN"
```
Comment (arvindsree) [inline test-infra/cmdExportOutput.sh]: why remove this?

Comment (zhiqiangjin) [reply test-infra/cmdExportOutput.sh]: These should be no change to cmdExportOutput.sh. Will change back.

File: aviatrix/resource_aviatrix_aws_tgw.go [hunk 2]
```diff
@@ -88,6 +88,24 @@ func resourceAviatrixAWSTgw() *schema.Resource {
 										Default:     "",
 										Description: "Customized Spoke VPC Routes. It allows the admin to enter non-RFC1918 routes in the VPC route table targeting the TGW.",
 									},
+									"subnets": {
+										Type:        schema.TypeString,
+										Optional:    true,
+										Default:     "",
+										Description: "",
```
Comment (arvindsree) [inline aviatrix/resource_aviatrix_aws_tgw.go]: Add description with example. Is it comma separated cidr as string?

Comment (arvindsree) [reply aviatrix/resource_aviatrix_aws_tgw.go]: why is default: empty? maybe we need computed: true

**Comment by arvindsree** [comment]
What happens if subnet is empty, does it show diff?

**Review by arvindsree — APPROVED** [review]

**Pull request merged**